Dockerfile

```diff
@@ -38,26 +39,30 @@ RUN yarn install --check-files
 COPY . .
 
 RUN bundle exec rake assets:precompile && \
-    rm -rf tmp/* log/* node_modules /tmp/*
+    rm -rf tmp/* log/* /tmp/*
+
+# Uses .yarnclean to filter node_module deletion 
+# https://classic.yarnpkg.com/lang/en/docs/cli/autoclean/
+RUN yarn autoclean -F 
```

Comment:
If you remove the files in a separate build step, then they will still be in the previous docker image layer and the image size will not decrease

Reply:
@saliceti We've locked the Alpine disto version to 3.19 now in the Dockerfile.